pages/windows/cmd.md

```diff
@@ -7,17 +7,17 @@
 
 `cmd`
 
-- Execute a [c]ommand:
+- Execute specific [c]ommands:
 
-`cmd /c "{{command}}"`
+`cmd /c "{{echo cmd is executed}}"`
```

Comment:
```suggestion
`cmd /c "{{echo Hello World}}"`
```

Reply:
I don't use windows, does this need to be in quotation marks since it is more than one word?

Reply:
They are not required (tested in wine cmd).